ts/torch_handler/object_detector.py

```diff
@@ -91,8 +97,9 @@ def handle(data, context):
 
         data = _service.preprocess(data)
         data = _service.inference(data)
-        data = _service.postprocess(data)
-
-        return data
+        if data:
+            return _service.postprocess(data)
+        else:
+            return ["No object detected"]
```

Comment:
Wondering if returning an empty list is more appropriate that returning an element a single element with string val.

Reply:
I am open to both ideas. Can remove the if-else block if you want .

Reply:
Agreed with @dhanainme

Reply:
Done.